src/main/resources/application.yml

```diff
@@ -0,0 +1,34 @@
+## System.properties ##
+server:
+  port: 8086
+  shutdown: graceful
+management:
+  server:
+    port: 8081
+
+logging:
+  level:
+    root: INFO
+
+## Hosts ##
+endpoint:
+  url: https://losore-tlg-api.apps.ocp-ut.regsys.brreg.no
```

Comment:
Dette er et testmiljø eksterne ikke vil ha tilgang til. I utgangspunktet tenkte jeg vi kunne ha en profile som peker mot ppe og difi Ver2, og en som peker mot produksjon og difi prod, slik at det blir enda enklere for eksterne.

Reply:
ref kommentarene nedenfor bør denne være del av application-test.yml og application-prod.yml med tilhørende verdier